installer/GNUmakefile

```diff
@@ -24,13 +24,13 @@ SIGNED_list  ?= tgl tgl-h
 ALIAS_OTHER_KEY_list += adl adl-s rpl rpl-s
 
 # Not supported in the main branch anymore, go to stable-v2.3
-# UNSIGNED_list += bdw
+# UNSIGNED_list += byt bdw cht hsw
```

Comment:
I wonder if we could start with one platform and get the merged. This is getting bewildering to review. Here it seems we are adding back support in middle of the series.

Reply:
It's a comment. It's true, I wanted to move it back to not remove them in the first place, but forgot. It has a comment in line 26 that these are the platforms kept in 2.3, so no reason to remove them